code/game/objects/effects/glowshroom.dm

```diff
@@ -153,8 +153,16 @@
 
 /obj/structure/glowshroom/attackby(obj/item/I, mob/user)
 	..()
-	if(I.damtype != STAMINA)
-		endurance -= I.force
+	var/force = I.force
+	if(istype(I, /obj/item/weapon/scythe))
+		var/obj/item/weapon/scythe/S = I
+		if(S.extend)	//so folded telescythes won't get damage boosts / insta-clears (they instead will instead be treated like non-scythes)

```

Comment:
Won't this mean that folded telescythes will simply provide no message and do nothing when used?